src/cuga/backend/tools_env/registry/mcp_manager/adapter.py

```
@@ -268,25 +275,25 @@ def _is_required(name: str, req: List[str]) -> bool:
-            out[prefix] = (py_t, default)
+            out[prefix] = spec
         else:
             # top-level
-            out.update({"body": (py_t, default)})
+            out.update({"body": spec})
         return
 
     if t == "array":
         # (Behavior unchanged; arrays remain a leaf with List[item_type].)
         item_schema = core.get("items") or {}
         item_core, item_nullable = _normalize_union(item_schema, resolve_ref)
```

Ruff (0.15.10)

[warning] Unpacked variable `item_nullable` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)